Dockerfiles/gadget-core.Dockerfile

```diff
@@ -42,26 +42,18 @@ RUN cd /gadget/gadget-container && \
 # Main gadget image
 FROM ${BASE_IMAGE}
 
-# install runtime dependencies  according to the package manager
-# available on the base image
 RUN set -ex; \
-	if command -v tdnf; then \
-		tdnf install -y libseccomp wget util-linux; \
-	elif command -v yum; then \
-		yum install -y libseccomp wget util-linux; \
-	elif command -v apt-get; then \
-		apt-get update && \
-		apt-get install -y seccomp wget util-linux; \
-	elif command -v apk; then \
-		apk add gcompat libseccomp wget util-linux; \
-	fi && \
```

Comment:
I think it'll break if a base image that doesn't provide the utilities needed in entrypoint.sh is used (#801).